airbyte-integrations/connectors/source-google-ad-manager/source_google_ad_manager/spec.yaml

```
@@ -0,0 +1,72 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/google_ad_manager
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Google Ad Manager Source Spec
+  type: object
+  required:
+    - type
+    - customer_name
+    - token_uri
+    - auth_provider_x509_cert_url
+    - auth_uri
+    - project_id
+    - private_key_id
+    - private_key
+    - client_id
+    - client_email
+  properties:
+    customer_name:
+      type: string
+      description: The name of the customer to connect to.
+      examples:
+        - "exported_customer"
```

```suggestion
    customer_name:
      title: Customer Name
      type: string
      description: The name of the customer to connect to.
      examples:
        - "exported_customer"
```
Add title to all params please.